backend/backend.go

```diff
@@ -110,3 +110,58 @@ func UnmarshalHistoryEvent(bytes []byte) (*HistoryEvent, error) {
 	}
 	return e, nil
 }
+
+func GetSubOrchestrationInstances(ctx context.Context, be Backend, iid api.InstanceID, includeAll bool) ([]api.InstanceID, error) {
```

Comment:
How does this method guarantee consistency of the data while performing database operations? I don't see a transaction being used, please confirm

Reply:
Updated the implementation that this method just loads state once using `backend.GetOrchestrationRuntimeState`. So there shouldn't be any issue in fetching the orchestration state. Please review again

Reply:
Modified impl